pint_server/alembic.ini

```diff
@@ -0,0 +1,89 @@
+# A generic, single database configuration.
```

Comment:
Since this file needs to be customised for people to be able to use alembic, but we don't want people to commit any changes they make to the file, perhaps we should switch to a workflow like the following:

1) We maintain an alembic.ini.example in the git repo, with an explicit .gitignore entry to ignore the alembic.ini
2) We document that people should copy the alembic.ini.example to alembic.ini and customise it as needed (this significantly reduces the change that someone inadvertently commits their creds)

Potentially we can have any dev-env setup scripting do the copy operation for us...

Reply:
Done.